### DISCUSSION TITLE
process an converted mf4 file with gdb

### DISCUSSION THREAD

**Discussion by emonjak** [danielhrisca/asammdf#903] [Q&A]
I opened a one file .mf4 from examples of Fa. CSS Electronics by asammdf and exported it as a csv file.
After it I converted this csv file to mf4 file and opened it by the same asammdf.
The asammdf with original file shows "Bus Logging" in main window and allows process it with dbc file, but don't do it with converted file.

Do I anything wrong?
I have append original, csv and converted files.

[test.zip](https://github.com/danielhrisca/asammdf/files/12498884/test.zip)
![original_file](https://github.com/danielhrisca/asammdf/assets/110918466/22d2be83-56ee-4279-aa00-9af2130eef4c)
![converted_file](https://github.com/danielhrisca/asammdf/assets/110918466/a05147f3-613f-4713-9c01-d49b101191fd)

**Comment by danielhrisca** [comment]
There is additional metadata related to the bus logging in the original file. The CSV does not have it

**Comment by emonjak** [comment]
@danielhrisca 
Thank you for response. 
Is it possible in asammdf to combine the such raw logged data with dbc/ldf to plot the signals?
